Makefile

```diff
@@ -8,14 +8,17 @@ presubmit: verify test licenses vulncheck ## Run all steps required for code to
 test: ## Run tests
 	go test ./... \
 		-race \
+		-timeout 20m \
```

Comment:
I'm guessing that this is because we've essentially doubled our cluster state suite tests, right? Can we add a TODO here when we remove all the machine suite tests here? Or should we keep this here in general to alert us when our tests are taking too long?

Reply:
Makes sense. I can add a TODO here